stage_jars.sh

```diff
@@ -37,8 +37,13 @@ function deploy_staging_jars {
     echo "Jars should not be created for SNAPSHOT versions. Use 'update_branch_version.sh' first."
     exit 1
   fi
-  flink_minor_version=$(echo ${FLINK_VERSION} | sed "s/.[0-9]\+$//")
-  version=${project_version}-${flink_minor_version}
+
+  if [ -z "${FLINK_VERSION}" ]; then
```

Comment:
Empty `${FLINK_VERSION}` works properly in the command line below 
`${MVN} clean deploy ${options} -Dflink.version=${FLINK_VERSION}`